src/cl/op-node/launcher.star

```diff
@@ -253,6 +256,18 @@ def get_service_config(
             "--sequencer.l1-confs=2",
         ]
 
+    if conductor_params:
+        cmd += [
+            "--conductor.enabled={0}".format("true"),
```

Comment:
just checking: shouldn't this use some conductor_params.enabled or something?
Otherwise, might as well lose the format call

Reply:
the `enabled` is only present on the input side, if set to `false` then the whole object becomes `None`. But true I'll just drop the format call